pages/api/check-verification-status.ts

```diff
@@ -19,12 +24,12 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
 
     const data = await result.json();
     if (data.status === "1") {
-      return res.status(200).json({ result: data.result });
+      return NextResponse.json({ result: data.result }, { status: 200 });
     } else {
-      return res.status(200).json({ result: data.result });
+      return NextResponse.json({ result: data.result }, { status: 200 });
```

Comment:
no cache in the case that it's not verified (user may verify immediately after and we don't want to hit cache in that case)